charts/exivity/templates/secret.yaml

```diff
@@ -35,5 +34,8 @@ metadata:
     app.kubernetes.io/component: proximity-api
     {{- include "exivity.labels" $ | indent 4 }}
 data:
-  license.key: {{ .Values.license | b64enc }}
-{{ end }}
+  {{- if (and .Release.IsUpgrade (not .Values.license)) }}
+  license.key: {{ index (lookup "v1" "Secret" .Release.Namespace (printf "%s-license-key" (include "exivity.fullname" $))).data "license.key" }}
+  {{- else }}
+  license.key: {{ .Values.license | required "A license is required to be set on inital install" | b64enc }}
```

Comment:
Small typo (inital vs initial). What is the reason we are requiring a licence now?

Reply:
The reason for requiring the license is because when trying to update it through the UI will fail as you cannot modify a config map from a volume mount. Not having a license results in a non functional install, you can still get to this state by entering a non-valid license. Fixing the typo now.